python/tests/server/test_http.py

```diff
@@ -450,6 +450,7 @@ def test_prediction_idempotent_endpoint_conflict(client, match):
         json={"input": {"sleep": 1}},
         headers={"Prefer": "respond-async"},
     )
+    time.sleep(0.001)
```

Comment:
Hmmmm. What's going on here?

Reply:
when I added it the test would fail because the second request would come in too quickly, but I think maybe the limiter change from async routes fixes it